resources/core/charts/service-catalog/charts/binding-usage-controller/templates/cluster-role.yaml

```diff
@@ -13,12 +13,12 @@ rules:
   verbs: ["*"]
 - apiGroups: ["apps"]
   resources: ["deployments"]
-  verbs: ["get", "list", "watch", "patch"]
+  verbs: ["get", "list", "watch", "patch", "update"]
 - apiGroups: ["kubeless.io"]
   resources: ["functions"]
   verbs: ["get","list","watch", "update"]
 - apiGroups: ["servicecatalog.kyma.cx"]
-  resources: ["servicebindingusages"]
+  resources: ["servicebindingusages", "usagekinds"]
```

Comment:
now we are giving full access to operating on sbu and usagekinds, can we restrict it to read, watch and maybe list?